frontend/src/app/features/assistance-selection/assistance-selection.component.html

```diff
@@ -0,0 +1,10 @@
+<h3>Select Your Desired Assistance Level on the Test:</h3>
+<div>
+  <ng-container *ngFor="let level of levels">
+    <button mat-raised-button color="primary"
+     (click)="handleSelectLevel(level)"
```

Comment:
Can we move the click below the `[id]` and `class` attributes? I generally prefer "active" code last.

Reply:
`handleSelectLevel` should pass `level.key` as the `grade-level-selection` did.